### PR TITLE
TE-735 clean up web-app

### PR DESCRIPTION
**Pull request by kbayraktar**
clean up old structure, 
created a new directory "login_test" with login page for our Quick Start Guide

### CODE REVIEW

File: rcp/org.testeditor.web.testexecutionlogview/login_test/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (kpanier) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/index.html]: I thought we would use an angular app to have some logic for test

File: rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
+@import url(http://fonts.googleapis.com/css?family=Vibur);
```
Comment (kpanier) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css]: Can we get this styles and the other files in a build step with npm?

File: rcp/org.testeditor.web.testexecutionlogview/login_test/index.html (hunk 1 above)
Comment (kbayraktar) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/index.html]: For the Quickstart Guide we just needed a HTML Form to present the IDs just to recognize the Web-Elements and enter some input. This is the reason why we choosed this approach for the first step. 
For a further step we should communicate about what functionality logic we want for automation testing and should build a separate angular application.

File: rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css (hunk 2 above)
Comment (kbayraktar) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css]: That would be possible, but i changed some parts of the original one to serve our needs. So the result would not be the same.

File: rcp/org.testeditor.web.testexecutionlogview/login_test/index.html (hunk 1 above)
Comment (kbayraktar) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/index.html]: I created a separate ticket for [TE-481|that https://testeditor.atlassian.net/browse/TE-481]

File: rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css (hunk 2 above)
Comment (kpanier) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css]: What was our need?

File: rcp/org.testeditor.web.testexecutionlogview/login_test/index.html (hunk 1 above)
Comment (kpanier) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/index.html]: ok, works for me

File: rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css (hunk 2 above)
Comment (kbayraktar) [inline rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css]: That we do not access websites through javascript where code will be executed without our permission.

**Comment by franzbecker** [comment]
@kpanier @kbayraktar What shall we do with this PR?

**Pull request merged**